### PR TITLE
Fixes related to planar reflection probes.

### PR DESCRIPTION
**Pull request by anisunity**
- Making the planar probe atlas's format match the color buffer's format.
- Removing the planarReflectionCacheCompressed setting from asset.

image

PlanarReflFormat

Tested while changing the color buffer format on the HDRP asset
Did not run the test on yamato, will probably require a screenshot update due to minor compression issues

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/HDBakedReflectionSystem.cs [hunk 1]
```diff
@@ -635,7 +635,7 @@ internal static void ImportAssetAt(HDProbe probe, string file)
                         importer.sRGBTexture = false;
                         importer.filterMode = FilterMode.Bilinear;
                         importer.mipmapEnabled = false;
-                        importer.textureCompression = hd.currentPlatformRenderPipelineSettings.lightLoopSettings.planarReflectionCacheCompressed
+                        importer.textureCompression = false
```
Comment (panaman67) [inline com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/HDBakedReflectionSystem.cs:638]: Could just set to uncompressed right?

Comment (anisunity) [reply com.unity.render-pipelines.high-definition/Editor/Lighting/Reflection/HDBakedReflectionSystem.cs:638]: woops didn't see the part of the lines after, my bad

**Comment by anisunity** [comment]
Tested while changing the color buffer format on the HDRP asset

**Comment by sebastienlagarde** [comment]
Please @JordanL8  keep reviewing the documentation, thanks :)

**Pull request merged**